include/stddef.h

```diff
@@ -99,4 +99,24 @@
  * Reference: Opengroup.org
  */
 
+/* Type whose alignment is supported in every context and is at least
+ * as great as that of any standard type not using alignment specifiers.
+ */
+
+typedef struct
```

Comment:
it's simpler to use union

Reply:
This implementation is follow gcc's version:
https://github.com/gcc-mirror/gcc/blob/master/gcc/ginclude/stddef.h#L408-L428